src/date.rs

```diff
@@ -629,6 +629,86 @@ impl Date {
         }
     }
     // endregion: checked arithmetic
+
+    // region: saturating arithmetic
+    /// Computes `self + duration`, saturating value on overflow.
+    ///
+    /// ```rust
+    /// # use time::{Date, ext::NumericalDuration, macros::date};
+    ///
```

Comment:
I didn't notice this until afterwards in your last PR, but let's remove the blank line here. rustdoc will otherwise show a leading blank line in docs, which isn't ideal.

Reply:
Okay